src/main/java/org/spongepowered/common/data/key/KeyRegistry.java

```diff
@@ -127,6 +127,9 @@ private static void generateKeyMap() {
         keyMap.put("spawner_minimum_delay", makeSingleKey(Short.class, MutableBoundedValue.class, of("SpawnerMinimumDelay")));
         keyMap.put("connected_directions", makeSetKey(Direction.class, of("ConnectedDirections")));
         keyMap.put("connected_north", makeSingleKey(Boolean.class, Value.class, of("ConnectedNorth")));
+        keyMap.put("connected_south", makeSingleKey(Boolean.class, Value.class, of("ConnectedSouth")));
+        keyMap.put("connected_east", makeSingleKey(Boolean.class, Value.class, of("ConnectedEast")));
+        keyMap.put("connected_west", makeSingleKey(Boolean.class, Value.class, of("ConnectedWest")));
```

Comment:
For serialisation this probably seems odd - maybe need to change datacontainer to only output ConnectedDirections instead of all five?